frontend/src/app/utils/smartyStreets.ts

```diff
@@ -55,3 +55,34 @@ export const getBestSuggestion = async (
     console.error("Unable to validate address:", error.message);
   }
 };
+
+function toLowerStripWhitespace(s: string | null): string {
```

Comment:
Kind of a nitpick, but I think this function should go into `frontend/src/app/utils/text.ts`

Reply:
Done in 6b18878c